src/main/java/uk/gov/digital/ho/hocs/domain/queue/migration/MigrationService.java

```diff
@@ -78,6 +80,13 @@ public void createMigrationCase(String messageId,
             caseId = createMigrationCaseResponse.getUuid();
             stageId = createMigrationCaseResponse.getStageId();
             log.info("Created migration case {}", createMigrationCaseResponse.getUuid());
+        } catch (HttpClientErrorException.Conflict e) {
```

Comment:
You've moved the status update to the top level in the queue listener, unsure if thats required? I think we should follow the existing practice of updating and just early returning? I like the completeWithStatus idea though!

Thoughts?